### ISSUE TITLE
Person Profile Creation Page

### ISSUE THREAD

**Issue by uoa-lchu090** [se701team3/Forgettable#24]
## Description
 
Front end design for users that want to create a new person profile in our web app. Page will be loaded in if a new person is being created. Prompts for information input will show and the data is sent to the database when created.
## Acceptance Criteria

- [ ] Page design is consistent with the settled on wireframe and UI design.
- [ ] All the necessary info needed for creating a new person is present.
- [ ] Users can easily input the needed info, it is clear where the information will need to be inputted.
- [ ] Newly created oersons will successfully appear in the database after created.

## Implementation details

Will be initialised once the frontend setup is done and the UI design has been finalised. Functionality implementation will be done when
the API and database have been sorted. 

Note that extra "nice to have" functionalities such as profile pics will need to be implemented here in the future.
## Testing notes

n/a

**Comment by lin8231** [comment]
Closing duplicate issue.
